src/release.sh

```diff
@@ -14,12 +14,16 @@ NPM_INFO="undefined"
 # instead of using -v use -n to check for an empty strings
 # -v is not working well on bash 3.2 on osx
 PATTERNFLY_DEPS="undefined"
+FEC_DEPS="undefined"
```

Comment:
Since we're including API clients too now:
```
RH_CLOUD_SERVICES_DEPS="undefined"
```

Reply:
Can you elaborate a bit? Do you want them to be separate? The packages are released under the same org. Both clients and frontend components.

Reply:
Oh no, I just thought the name should be changed to `RH_CLOUD_SERVICES_DEPS`, since that includes the FEC and API client versions

Reply:
Right gotcha :smile: will do.